lib/metadata/in_memory/backend.js

```diff
@@ -41,7 +42,8 @@ const metastore = {
             if (!metadata.buckets.has(bucketName)) {
                 return cb(errors.NoSuchBucket);
             }
-            return cb(null, metadata.buckets.get(bucketName));
+            return cb(null, BucketInfo.fromObj(
```

Comment:
By doing `git blame` I saw that this was added as a backward compatibility for the splitter. Are we not going to support the old splitter usage at all?